src/build.js

```diff
@@ -11,6 +12,8 @@ import { setCwd } from './plugins/npm-plugin/registry.js';
 export default async function build(options = {}) {
 	options.out = options.out || 'dist';
 
+	await premove(options.out, { cwd: options.cwd });
```

Comment:
I think `out` is always absolute path because of `nornalize-output` so technically the `cwd` option isn't needed but can't hurt... basically a noop

Reply:
Jk